modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc

```diff
@@ -0,0 +1,105 @@
+
+= Creating Client Applications with Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: grant access, api, contracts
+
+
+For automation use cases, you can use Connected Apps to create and manage client applications.
+
+Connected Apps help user delegate their access without sharing sensitive credentials or giving full control of their applications to third parties. Actions taken by connected apps are audited, and users can revoke access at any time.
+
+
+== Creating a Connected App
+
+To create a Connected App, see xref:access-management::connected-apps-developers.adoc[].
+
+When you create the Connected App to manage client applications, ensure that you:
+
+. Select *App acts on its own behalf (client credentials)* for *Type*.
+. Add the *Application Creator* scope.
+
+image::connected-app-client-apps.png[Screenshot calling out the location of App acts on its own behalf (client credentials) and Application Creator]
```

Comment:
I agree that the screenshot is unnecessary. It doesn't really add value, because it doesn't actually show how to get to that modal. It also doesn't show any existing scopes, so doesn't show anything that someone can't easily follow in the text.